From fae22c2f1bf5d40b0ba5cb47f597daaf3ed4ae6b Mon Sep 17 00:00:00 2001
From: Richard Hughes <richard@hughsie.com>
Date: Mon, 17 Mar 2014 19:35:04 +0000
Subject: [PATCH] Do not install into python_sitelib

The .pyc and .pyo files are actually architecture specific and this causes
PackageKit to fail the RHEL multilib self tests.
---
 configure.ac               | 11 +++++++++--
 contrib/PackageKit.spec.in |  5 +++--
 2 files changed, 12 insertions(+), 4 deletions(-)

diff --git a/configure.ac b/configure.ac
index 68ce26a..8cdabd7 100644
--- a/configure.ac
+++ b/configure.ac
@@ -91,8 +91,14 @@ if test x$enable_python3 = xyes; then
 else
 	AM_PATH_PYTHON([2.7])
 fi
-
-PYTHON_PACKAGE_DIR=${pythondir}/packagekit
+AC_ARG_WITH([python_package_dir],
+	    AS_HELP_STRING([--with-python-package-dir=<location>],
+			   [Location for python modules]))
+if test -z "$with_python_package_dir" ; then
+	PYTHON_PACKAGE_DIR=${pythondir}/packagekit
+else
+	PYTHON_PACKAGE_DIR=${with_python_package_dir}/packagekit
+fi
 AC_SUBST(PYTHON_PACKAGE_DIR)
 
 if test "$GCC" = "yes"; then
